crates/sncast/src/helpers/token.rs

```diff
@@ -0,0 +1,61 @@
+use anyhow::{Result, anyhow};
+use clap::ValueEnum;
+use conversions::serde::serialize::{BufferWriter, CairoSerialize};
+use serde::{Deserialize, Serialize};
+use starknet::macros::felt;
+use starknet_types_core::felt::Felt;
+use std::{fmt, str::FromStr};
+
+const STRK_CONTRACT_ADDRESS: Felt =
+    felt!("0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d");
+
+const ETH_CONTRACT_ADDRESS: Felt =
+    felt!("0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7");
+
+#[derive(Default, Serialize, Deserialize, Clone, Copy, Debug, PartialEq, Eq, ValueEnum)]
```

Comment:
```suggestion
#[derive(Default, Serialize, Clone, Copy, Debug)]
```

It is enough. This is internal code, if somebody will need them they will just derive it